python/lsst/daf/base/propertyContainer/propertyContainerContinued.py

```diff
@@ -47,7 +45,10 @@ def _propertyContainerElementTypeName(container, name):
 
 
 def _propertyContainerGet(container, name, asArray=False):
-    """Extract a single Python value of unknown type"""
+    """Get a value of unknown type as a scalar or array
+
+    Return a scalar if there is only one value, else return an array
```

Comment:
I see no evidence that the asArray argument is used by any code in our stack, except two unit tests in daf_base, so I am going to try removing it from `PropertySet.get` and `PropertyList.get` and see if Jenkins passes. If it does, this will make it much easier to implement shared base code for `getArray` and `getScalar` since we can change the argument to `asScalar` and only return a scalar if that is true.

Reply:
[DM-14770](https://jira.lsstcorp.org/browse/DM-14770) provide an example of why the `asArray` argument may be good to delete. I found code that treats the second argument as a default value, in case the requested item is missing.